### DISCUSSION TITLE
GitHub Pages

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [sam-astro/Astro8-Computer#23] [General]
*Just want to share my progress with @sam-astro*
*regarding the GitHub Pages site.*

Preview : https://electronicsarchiver.github.io/Astro8-Computer/

<br>

## Notes

- The page should be hosted from a different branch

- You can already create the branch from your current main branch

- No need to enable github pages until I'm done

  Once I'm done enable, it in the repo settings and use said branch + `root`

**Comment by ElectronicsArchiver** [comment]
Unfortunately, jerkyl or the theme don't allow `<kbd>` tags 
that I used for syntax visualization, I'll go with this instead:

```
define 〈 Ａｄｄｒｅｓｓ 〉 〈 Ｖａｌｕｅ 〉
```

**Comment by ElectronicsArchiver** [comment]
I wanted to open a PR, however I need you to create
a branch first, `Documentation` for example.

**Reply by sam-astro** [reply]
Made branch Documentation
